### DISCUSSION TITLE
Move this help/man off of GitHub to a Wiki not on the Dark Web

### DISCUSSION THREAD

**Discussion by 62f** [keepassxreboot/keepassxc#9929] [Ideas]
Since TOTP Authentication is now being required of GitHub contributors, it's presumably unobtanium to people needing to use Authenticators, of which keeppassxc is one.
Get rid of the catch-22 scenarios and circular logic. Offer this guide and faqs somewhere else that's not totally and hopelessly hidden behind 2FA and SSA certificates.
[Linux.org](Linux.org) comes to mind.

**Comment by droidmonkey** [comment]
You can access the wiki as a completely anonymous user...

**Reply by 62f** [reply]
Yésican and Yésidid. Indeed there's a support WiKi which links right back to GitHub, but its last update was in 2021, predating this new 2FA requirement: which I believe is how I ended up here in the first place, if indeed that stale thing was the same wiki to which you allude.

**Reply by droidmonkey** [reply]
I have no idea what you are referring to, to be honest. We have nothing to do with GitHub's decision to use 2FA. Our documentation is accessible from within the app (offline) or here: https://keepassxc.org/docs/KeePassXC_UserGuide
